### PR TITLE
Fix tags and keys escaping

### PR DESCRIPTION
**Pull request by mmatloka**
See https://github.com/iZettle/dropwizard-metrics-influxdb/issues/45
and
https://docs.influxdata.com/influxdb/v0.13/write_protocols/line/#key

### CODE REVIEW

**Comment by mmatloka** [comment]
@ngong please also look at this PR :)

File: metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/utils/InfluxDbWriteObjectSerializer.java [hunk 1]
```diff
@@ -84,8 +84,8 @@ private void formattedTime(Long time, TimeUnit precision, StringBuilder stringBu
     }
 
     private String escapeKey(String key) {
-        String toBeEscaped = SPACE.matcher(key).replaceAll("\\ ");
-        toBeEscaped = COMMA.matcher(toBeEscaped).replaceAll("\\,");
+        String toBeEscaped = SPACE.matcher(key).replaceAll("\\\\ ");
+        toBeEscaped = COMMA.matcher(toBeEscaped).replaceAll("\\\\,");
         return EQUAL.matcher(toBeEscaped).replaceAll("\\=");
```
Comment (ngong) [inline metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/utils/InfluxDbWriteObjectSerializer.java]: I think EQUAL here should do the same.

**Comment by ngong** [comment]
@mmatloka nice one, could you do a rebase?

File: metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/utils/InfluxDbWriteObjectSerializer.java (hunk 1 above)
Comment (mmatloka) [inline metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/utils/InfluxDbWriteObjectSerializer.java]: hmm, I'am looking at the docs https://docs.influxdata.com/influxdb/v0.13/write_protocols/line/#key and it seems that `=` is even not allowed in keys, only spaces and commas.

Comment (ngong) [inline metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/utils/InfluxDbWriteObjectSerializer.java]: It makes sense not to include `=` in the keys, then we should remove `=` from the keys

**Comment by mmatloka** [comment]
Corrected, squashed and rebased.

**Comment by ngong** [comment]
I finally find it in the document. Equal mark is escaped both for key and value.
[](https://docs.influxdata.com/influxdb/v0.13/write_protocols/write_syntax/)
I will merge this for now, and fix the equal mark issue.

**Comment by mmatloka** [comment]
@ngong where is it described?

**Comment by ngong** [comment]
[https://docs.influxdata.com/influxdb/v0.13/write_protocols/write_syntax/](https://docs.influxdata.com/influxdb/v0.13/write_protocols/write_syntax/)
Sorry, fail to create a link, LOL. I'll go check all syntax one by one which are described in that doc

**Pull request merged**